order/views.py

```diff
@@ -1,3 +1,97 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from user.models     import User
+from product.models  import Product, Brand, ProductGroup, MainCategory, SubCategory, SubSubCategory, WatchList
+from .models         import OrderItem, Order, OrderStatus, Shipment, CartItem
+
+from user.utils import check_user
+
+class CartView(View):
+    @check_user
+    def post(self, request):
+        data    = json.loads(request.body)
+
+        try:
+            product_id      = data['product_id']
+            quantity        = data['quantity']
+        
+        except KeyError:
+            return JsonResponse({'message': "KEY_ERROR"}, status = 400)
+        
+        exist_product = CartItem.objects.filter(product_id = product_id, user_id = user_id)
```

Comment:
CartItem.objects.filter() 함수가 실행되면 CartItem 인스턴스의 집합(QuerySet[<CartItem 1>, <CartItem 2>,.. ])으로 반환되는데, exist_product라고 하는 변수에 할당하고 있습니다.

변수를 명확하게 할당해주셔야 동료 개발자들이 봤을때, 혹은 내가 나중에 다시 이 코드를 봤을 때, 코드를 파악하기가 쉽습니다!
```suggestion
        cart_items = CartItem.objects.filter(product_id = product_id, user_id = user_id)
```

Reply:
네, 알겠습니다!